app/collections/show-list.cjsx

```diff
@@ -168,7 +176,16 @@ module.exports = React.createClass
       .then =>
         @props.collection.uncacheLink 'subjects'
 
-  deleteSubjects: () ->
+  confirmDeleteSubjects: ->
+    alert (resolve) =>
+      <div className="confirm-delete-dialog content-container">
+        <p>Are you sure you want to delete {@state.selected.length} subjects from this collection? This action is irreversible!</p>
```

Comment:
Let's make this wording consistent with the button. How about `remove` instead of `delete`?

Also, I don't think `This action is irreversible` is necessary. Seems a bit dramatic :)